### ISSUE TITLE
Click Event not firing in Chrome for sometime

### ISSUE THREAD

**Issue by mamallana** [CreateJS/EaselJS#792]
var boxClone = icon.clone();
 boxClone.on("click", function (evt) {  
alert("test");
}

Click event not working sometime in Chrome for the cloned bitmap image in EaselJs.

Please someone assist

**Comment by lannymcnie** [comment]
I did a quick test, and it looks fine:
http://jsfiddle.net/lannymcnie/6dn145j9/

```js
var bmp = new createjs.Bitmap(img).set({name:"bmp1"});
stage.addChild(bmp);
  
var bmp2 = bmp.clone().set({x:200, name:"bmp2"});
stage.addChild(bmp2);
stage.update();
  
bmp.on("click", handleClick);
bmp2.on("click", handleClick);

function handleClick(event) {
  	alert(["bmp clicked", event.target.name]);
}
```

Can you see if that works for you? If not, let me know the browser/OS, and any other details.

**Comment by mamallana** [comment]
Thanks a lot for the reply, i was thinking it may be because of the cross origin issue, and it was not happening all time, sometimes morning it will work and evening it won't work.  May be your code 

var img = document.createElement("img");
img.onload = init;
img.crossOrigin = "Anonymous";

Will help in that case.

**Comment by lannymcnie** [comment]
Yes, cross-origin will (when set up right) allow content loaded cross-domain to be interacted with (click uses pixel access to determine hit tests). This will affect your original Bitmap as well, so the cloning aspect of your bug is likely a red herring, and not the root issue.

**Comment by lannymcnie** [comment]
I am closing this out. Feel free to reopen if it is an actual bug, and not a CORS issue.
